Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -673,3 +725,9 @@ Vector<Match> HexEditor::find_all_strings(size_t min_length)
 
     return matches;
 }
+
+void HexEditor::reset_cursor_blink_state()
+{
+    m_cursor_blink_active = true;
+    m_blink_timer->restart();
+}
```

Comment:
Please set your editor to insert a newline at end of file

Reply:
Yeah, thanks, done :-) I just enabled the pre-commit checks.